Client/Frontend/Browser/SwipeAnimator.swift

```diff
@@ -64,13 +64,13 @@ extension SwipeAnimator {
         // Calculate the edge to calculate distance from
         let translation = velocity.x >= 0 ? CGRectGetWidth(container.frame) : -CGRectGetWidth(container.frame)
         let timeStep = NSTimeInterval(abs(translation) / speed)
+        self.delegate?.swipeAnimator(self, viewDidExitContainerBounds: self.animatingView)
```

Comment:
Since we've moved this before the animation the name of the delegate method doesn't make sense anymore. We should probably rename it to `viewWillExitContainerBounds` instead.